api-contracts/openapi/components/schemas/workflow.yaml

```diff
@@ -25,9 +25,13 @@ Workflow:
       items:
         $ref: "#/Job"
       description: The jobs of the workflow.
+    tenantId:
+      type: string
+      description: The tenant id of the workflow.
```

Comment:
api change?

Reply:
i just snuck this in to handle mismatch between the detail resource and the active tenant... i think its okay here since we actually can review it